### PR TITLE
Add deploy_development job to CircleCI

### PR DESCRIPTION
**Pull request by Reyko**

### CODE REVIEW

**Review by psoleckimoj — DISMISSED** [review]
LGTM!

File: .circleci/config.yml [hunk 1]
```diff
@@ -109,18 +109,23 @@ jobs:
         command: bundle -v
     - restore_cache:
         keys:
-        -  prison-visits-public-{{ checksum "Gemfile.lock" }}
+        - prison-visits-public-{{ checksum "Gemfile.lock" }}
+        - prison-visits-public-
```
Comment (psoleckimoj) [inline .circleci/config.yml:113]: Not sure this 2nd one is needed?

**Review by psoleckimoj — APPROVED** [review]
LGTM!

File: .circleci/config.yml (hunk 1 above)
Comment (Reyko) [reply .circleci/config.yml:113]: Hey Paul, this is used to restore any last partial cache that would match `- prison-visits-public-` 

https://circleci.com/docs/caching/#restoring-cache

**Pull request merged**